src/components/HomePage.tsx

```diff
@@ -155,41 +180,27 @@ export default class HomePage extends PureComponent<Props> {
       const message = "Couldn't proceed with authentication.";
       notifyError(message, error);
     }
-  };
+  }, []);
```

Comment:
This empty dependency array (and the one above) essentially makes this useEffect equivalent to `componentDidMount`.